cypress/integration/groups_page_test.js

```diff
@@ -12,20 +12,26 @@ describe('Groups page tests', () => {
         cy.fixture('groups')
           .as('groups')
           .then((groups) => {
-            cy.server();
-            cy.route('GET', `/appengine/v1/${realm.name}/groups`, groups);
             groups.data.forEach((groupName) => {
               const encodedGroupName = encodeURIComponent(groupName);
               const groupFixture = groupName.startsWith('special characters')
                 ? `group.special-characters.devices.json`
                 : `group.${groupName}.devices.json`;
               cy.fixture(groupFixture).as(`${encodedGroupName}-devices`);
-              cy.route(
+              cy.intercept(
                 'GET',
                 `/appengine/v1/${realm.name}/groups/${encodedGroupName}/devices?details=true`,
-                `@${encodedGroupName}-devices`,
+                { fixture: groupFixture },
               );
             });
+            // TODO: Since intercept() doesn't properly handle URLs with special characters, the
```

Comment:
is there an issue we can link to keep track of the current progress on cypress?

Reply:
Thanks for bringing it up. I looked deeper into the issue and it was my mistake, the URL to match wasn't properly encoded.
Now it's fixed.